### PR TITLE
Removes glide_size, for SMOOTHER MOVEMENT

### PR DESCRIPTION
**Pull request by praisenarsie**
:cl: 2016 called
code: Movement is smoother.
/:cl:

[why]: wonder what 2017 /tg/ think of this, will they hate it more or will they actually consider it. no, this is not obvious bait. i just want opinions on why players prefer our current movement rather a smoother alternative.

It's been over one year, people's opinions are subject to change. (no revert wars too please)

If this does not work, would it be possible to seek better movement that would feel less jarring for players. I am sick of people running at lightspeed.

tagging relevant people
@MrStonedOne @lzimann @optimumtact @KorPhaeron @Incoming5643 @Cyberboss @Fox-McCloud @Iamgoofball

### CODE REVIEW

**Comment by lzimann** [comment]
This "smooth" movement was horrible, it was not smooth at all.

**Comment by FlattestGuitar** [comment]
Obvious bait.

**Comment by RemieRichards** [comment]
image

**Comment by praisenarsie** [comment]
> This "smooth" movement was horrible, it was not smooth at all.

**Comment by praisenarsie** [comment]
i still want some movement that isn't jarring :/

**Comment by RemieRichards** [comment]
we have that, it's in game already.

**Comment by praisenarsie** [comment]
@RemieRichards let me rephrase, i still want some movement where you don't go mach 5

**Comment by RemieRichards** [comment]
well that's nothing to do with smoothness, that's just general speed.
you'd have to try and get a vote done over the movespeed set in the game config, which I don't think you'd win, unfortunately for yourself.

The movespeed used to be faster than it is now in fact, and only got in because it was forced through without player consent, people eventually learned to accept it but I doubt a lot of them would want it to get EVEN slower.
